test/kubemark/resources/hollow-node_template.json

```diff
@@ -27,10 +27,22 @@
 						}
 					},
 					{
+						"name": "npdconfig-volume",
+						"configMap": {
+							"name": "node-problem-detector-config"
+						}
+					},
+					{
 						"name": "logs-volume",
 						"hostPath": {
 							"path": "/var/logs"
 						}
+					},
+					{
+						"name": "npdlogs-volume",
```

Comment:
This volume is not for npd logs, but for logs that NPD is reading to detect whether the node is healthy.
We should make the name more explicit with the name here (maybe something like kernellog-volume or sth?)